apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -25,238 +28,245 @@ context('Actions', () => {
     cy.intercept('**/vosManager/createVo/**').as('createVo')
       .intercept('**/vosManager/getVos').as('getVos')
       .get('[data-cy=new-vo-button]')
-        .click()
+      .click()
       .wait('@getVos')
       .get('[data-cy=vo-name-input]')
-        .type('test-e2e-vo')
+      .type('test-e2e-vo')
       .get('[data-cy=vo-shortname-input]')
-        .type('test-e2e-vo')
+      .type('test-e2e-vo')
       .get('[data-cy=create-vo-button]')
-        .click()
+      .click()
       .wait('@createVo')
       .wait('@getVos')
```

Comment:
```suggestion

```
I know that you didnt wrote this, but on this place we shouldn't wait for getVos because after creating vo we are redirected to vo detail
Instead we should for getVoDetail